gtests/net/mptcp/fastopen/client-TCP_FASTOPEN_CONNECT-no-cookie.pkt

```diff
@@ -18,5 +20,9 @@
 +0.01     >  P.  501:1501(1000)  ack 1                <nop, nop, TS val 100 ecr 700, mpcapable v1 flags[flag_h] key[ckey, skey] mpcdatalen 1000, nop, nop>
 +0.01     <   .  1:1(0)          ack 1501  win 225                                  <dss dack8=1001 nocs>
 
+// Similar to TCP?
```

Comment:
just to be clear: here I want to check if it is similar to TCP where we should get a errno 22 (Invalid arg)

(it is not a question for the review, I first check what we had with TCP ;-) )